lightstep/resource_alert.go

```diff
@@ -37,3 +43,32 @@ func getQueriesFromUnifiedConditionResourceData(
 	}
 	return queries, nil
 }
+
+func getCompositeAlertFromUnifiedConditionResourceData(compositeAlertIn *client.CompositeAlert) (interface{}, error) {
+	subAlerts := make([]map[string]interface{}, 0)
+	for _, subAlertIn := range compositeAlertIn.Alerts {
+		queries, err := getQueriesFromUnifiedConditionResourceData(subAlertIn.Queries, subAlertIn.Name, subAlertIn.Name)
+		if err != nil {
+			return nil, err
+		}
+
+		subAlerts = append(subAlerts, map[string]interface{}{
+			"name":  subAlertIn.Name,
+			"title": subAlertIn.Title,
+			"expression": []map[string]interface{}{
+				{
+					"is_no_data": subAlertIn.Expression.IsNoData,
+					"operand":    subAlertIn.Expression.Operand,
+					"thresholds": []interface{}{
+						buildUntypedThresholdsMap(subAlertIn.Expression.Thresholds),
+					},
+				},
+			},
+			"query": queries,
+		})
+	}
+
+	return []map[string][]map[string]interface{}{{
+		"alert": subAlerts,
```

Comment:
I don't love the name `alert` here because I feel like it makes the terraform resource file weird to read, maybe `subalert`?